### PR TITLE
Worked example: multiple tags

### LINKED ISSUE #25: Change workedexample directive to support multiple tags
Right now the directive allows only one tag per worked example, however, there are some worked examples that contain more than one topic, for example, "Awards and Contracts" https://standard.open-contracting.org/latest/en/guidance/map/awards_contracts/ and https://standard.open-contracting.org/latest/en/guidance/map/buyers_suppliers/. We could change the directive to allow defining multiple tags separated by commas

### PR DESCRIPTION
**Pull request by yolile**
closes #25

### CODE REVIEW

**Comment by yolile** [comment]
A test is failing but I think that is for a non-related feature?

**Comment by jpmckinney** [comment]
I can't reproduce locally. I'm re-running all jobs.

**Comment by yolile** [comment]
I was able to reproduce the error locally, so I made a commit to fix it https://github.com/open-contracting/sphinxcontrib-opencontracting/pull/26/commits/32d044ab6ff2822b6de56df65eab7d668b0641ce. Now the error is not happening locally anymore but they are still here 🤷🏻‍♀️

**Comment by jpmckinney** [comment]
I think you were only able to reproduce because the nonreadable files were maybe left over on disk / not cleaned up by the test. The test's expectations are correct.

**Comment by jpmckinney** [comment]
I removed the commit and made a new commit to help with debugging CI (for whatever reason `pytest -vv` isn't printing the full warnings message)

**Comment by jpmckinney** [comment]
The test was expecting WARNING but now the result is ERROR. I'll check which versions of Sphinx, etc. are installed in CI.

**Comment by jpmckinney** [comment]
Yes, Sphinx 4.4 uses ERROR. Apparently this was a bug: https://www.sphinx-doc.org/en/master/changes.html#id45 https://github.com/sphinx-doc/sphinx/issues/9979

File: docs/index.rst [hunk 1]
```diff
@@ -123,13 +123,13 @@ Use:
 
 Where:
 
-- The ``workedexample`` directive is used at the top of a section to mark the content as a worked example. Its argument is used as the text of the hyperlink rendered by the ``workedexamplelist`` directive.
+- The ``workedexample`` directive is used at the top of a section to mark the content as a worked example. Use the ``tags`` attribute to indicate the content this worked example is referring to. Multiple values are allowed by using comma-separated values. Its argument is used as the text of the hyperlink rendered by the ``workedexamplelist`` directive.
```
Comment (jpmckinney) [inline docs/index.rst:126]:
```suggestion
- The ``workedexample`` directive is used at the top of a section to mark the content as a worked example. Its argument is used as the text of the hyperlink rendered by the ``workedexamplelist`` directive. Tag the worked example with one or more comma-separated tags, using the ``tags`` option.
```

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**